sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -16,6 +16,7 @@
 #include <errno.h>
 #include <stdbool.h>
 
+#include "kernel_defines.h"
```

Comment:
This is not needed right?

Reply:
Oh yeah, was from a previous iteration where I thought I could use `IS_ACTIVE()` (which apparently doesn't work with strings)

Reply:
Amended that change. I hope this was okay.

Reply:
Yeah, I was thinking about that. One way I can think of is having a `PSEUDOMODULE` to enable the feature and a second macro to pass the value. That way you can use C conditionals